src/testFixtures/java/uk/gov/di/ipv/cri/common/library/client/CommonApiClient.java

```diff
@@ -46,6 +52,33 @@ public HttpResponse<String> sendAuthorizationRequest(String sessionId)
         return sendHttpRequest(request);
     }
 
+    public HttpResponse<String> sendNewAuthorizationRequest(String sessionId)
+            throws IOException, InterruptedException {
+        var url =
+                new URIBuilder(this.clientConfigurationService.getPrivateApiEndpoint())
+                        .setPath(this.clientConfigurationService.createUriPath("authorization"))
+                        .addParameter(
+                                "redirect_uri",
+                                new URIBuilder("https://test-resources.review-a.dev.account.gov.uk")
```

Comment:
since this is common, do we want to be hard coding the url here? Specifically the environment and cri-id? For a common lib, I would expect this to be re-usable by others.